### ISSUE TITLE
dbt templater error when enabled: false

### ISSUE THREAD

**Issue by ciejer** [sqlfluff/sqlfluff#644]
When running across a dbt_ project using dbt templater, sqlfluff looks for the compiled version of every model in the repository.
If one of the models has been disabled in dbt_project.yml, there is no copy of it in the compilation folder; causing sqlfluff to error.

Steps to recreate:

1. Install sqlfluff in a dbt repository
2. Set `templater = dbt` in .sqlfluff
3. Add `enabled = false` in dbt_project.yml for one or more models
4. Run `sqlfluff fix models` in project directory

Full traceback:
```
Traceback (most recent call last):
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\jenkins\AppData\Local\Programs\Python\Python37\Scripts\sqlfluff.exe\__main__.py", line 7, in <module>
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1025, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 955, in main
    rv = self.invoke(ctx)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1517, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 1279, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\click\core.py", line 710, in invoke
    return callback(*args, **kwargs)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\cli\commands.py", line 413, in fix
    result = lnt.lint_paths(paths, fix=True, ignore_non_existent_files=False)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\core\linter.py", line 1278, in lint_paths
    ignore_files=ignore_files,
    target_file.read(), fname=fname, fix=fix, config=config
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\core\linter.py", line 1043, in lint_string
    parsed = self.parse_string(in_str=in_str, fname=fname, config=config)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\core\linter.py", line 797, in parse_string
    in_str=in_str, fname=fname, config=config
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\core\templaters\dbt.py", line 181, in process
    return self._unsafe_process(fname, in_str, config)
  File "c:\users\jenkins\appdata\local\programs\python\python37\lib\site-packages\sqlfluff\core\templaters\dbt.py", line 220, in _unsafe_process
    raise RuntimeError("File %s was not found in dbt project" % fname)
RuntimeError: File <removed filename as not relevant> was not found in dbt project
```

**Comment by pwildenhain** [comment]
Wow, great find! We should either fix this before 0.4.0, or quickly include it in a minor release 👌

**Comment by alanmcruickshank** [comment]
I believe this was fixes in #712

**Comment by panasenco** [comment]
@alanmcruickshank The other issue was closed because it was a duplicate of this one. There was no fix and I'm still experiencing the same bug when using sqlfluff from the GitHub master branch. Please reopen...
